### DISCUSSION TITLE
What do you think is the answer?

### DISCUSSION THREAD

**Discussion by batuhangithub** [enzifiri/GitHub-Achievements#375] [Q&A] [answered]
You want to find the even numbers within a list and store them in a new list. Let's say the content of the list is: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]. How would you identify the even numbers in this list and store them in a new list?

**Comment by muslumhanozturk** [comment, marked as answer]
# Original list
lst = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]

# Create an empty list to store even numbers
even_numbers = []

# Iterate through the list and add even numbers to the new list
for num in lst:
    if num % 2 == 0:
        even_numbers.append(num)

# New list containing even numbers
print(even_numbers)
